### DISCUSSION TITLE
Nonsense error message during deploy

### DISCUSSION THREAD

**Discussion by sillvva** [vercel/community#741] [Help] [answered]
I'm getting a completely nonsense error message when deploying my app on Vercel. It doesn't make any sense.

It started showing up a couple builds ago, but wasn't blocking deploy, so I ignored it. After adding Prisma to my app, it is now blocking deploys. It's the same error message, and doesn't seem to have anything to do with Prisma, but adding Prisma caused it to get blocked. The error stack seems to have more to do with Vercel.

> Using @sveltejs/adapter-vercel
--
20:10:38.778 | error during build:
20:10:38.778 | Error: Failed to parse /tmp/yarn--1657486265488-0.35481788128142555/node as script:
20:10:38.778 | Unexpected token (2:5)
20:10:38.779 | at Object.analyze [as default] (/vercel/path0/node_modules/.pnpm/@vercel+nft@0.20.1/node_modules/@vercel/nft/out/analyze.js:223:30)
20:10:38.779 | at Job.emitDependency (/vercel/path0/node_modules/.pnpm/@vercel+nft@0.20.1/node_modules/@vercel/nft/out/node-file-trace.js:281:52)
20:10:38.779 | at async /vercel/path0/node_modules/.pnpm/@vercel+nft@0.20.1/node_modules/@vercel/nft/out/node-file-trace.js:292:21
20:10:38.779 | at async Promise.all (index 3)
20:10:38.779 | at async Job.emitDependency (/vercel/path0/node_modules/.pnpm/@vercel+nft@0.20.1/node_modules/@vercel/nft/out/node-file-trace.js:287:9)
20:10:38.779 | at async /vercel/path0/node_modules/.pnpm/@vercel+nft@0.20.1/node_modules/@vercel/nft/out/node-file-trace.js:316:21
20:10:38.779 | at async Promise.all (index 6)
20:10:38.779 | at async Job.emitDependency (/vercel/path0/node_modules/.pnpm/@vercel+nft@0.20.1/node_modules/@vercel/nft/out/node-file-trace.js:287:9)
20:10:38.780 | at async /vercel/path0/node_modules/.pnpm/@vercel+nft@0.20.1/node_modules/@vercel/nft/out/node-file-trace.js:339:21
20:10:38.780 | at async Promise.all (index 21)

**Comment by swarnava** [comment, marked as answer]
Hi, Can you share your deployment url?

**Reply by sillvva** [reply]
https://vercel.com/sillvva/mattdekok-sveltekit/A9wdnwMWwPpadC8rWmaGdiAPvLkm

**Reply by sillvva** [reply]
This also seems to only happen on Vercel. If I build and preview locally, I get no errors.

**Reply by sillvva** [reply]
Any thoughts?

**Reply by sillvva** [reply]
OK then 😭

**Reply by swarnava** [reply]
Sorry about the delayed reply. Are you able to run the project locally?

**Reply by sillvva** [reply]
Yes. I have zero errors when I use the dev, build, and preview commands locally. This error only occurs on Vercel. 

I wonder if there is an issue with the Vercel adapter for Sveltekit. The error was there, but non-blocking until Prisma was added. Perhaps an issue between the Vercel adapter and Prisma started blocking? Not sure.

**Reply by sillvva** [reply]
For example, here is a build where the error exists, but the build successfully completed. This was after I redeployed without Prisma.
https://vercel.com/sillvva/mattdekok-sveltekit/AeZSrkAphkoULrXCdfss5fFeS5hw

**Reply by swarnava** [reply]
I will recommend creating a support request so that our team can take a look. Please provide a minimal steps so team can look into this.

**Reply by sillvva** [reply]
Support case opened and provided with a link to the commit that first started blocking, and the deployment URLs that failed and succeeded. Not sure what else I can provide.
